p11kit/p11kit.go

```diff
@@ -250,22 +250,24 @@ type session struct {
 	slotID uint64
 }
 
-func (s *session) attributeValue(objectID uint64, tmpl []attributeTemplate) ([]attribute, error) {
+func (s *session) attributeValue(objectID uint64, tmpl []attributeTemplate) ([]attribute, bool, error) {
```

Comment:
can you add a comment here explaining what the bool represents? e.g.

```
// attributeValue looks up the values for the attribute templates, and returns if all the values were found or not.
// If not, the client sends the returned attributes, but also the error code CKR_ATTRIBUTE_TYPE_INVALID.
```

Reply:
Done